src/pages/LendPage/LendDetailPage.tsx

```diff
@@ -0,0 +1,450 @@
+              transform='translate(-3.744 -4.167)'
+              fill='#448aff'
+            />
+          </g>
+        </svg>
+        <Box fontSize={'24px'} fontWeight={'bold'} color={'white'}>
+          Quickswap Pool
+        </Box>
+        <Box display={'flex'} gridGap={'2px'}>
+          <USDTIcon size={'24px'} />
+          <USDTIcon size={'24px'} />
+          <USDTIcon size={'24px'} />
+          <USDTIcon size={'24px'} />
+          <USDTIcon size={'24px'} />
+          <Box
+            bgcolor={'#32394c'}
+            width={'24px'}
+            height={'24px'}
+            borderRadius={'100%'}
+            display={'flex'}
+            alignItems={'center'}
+          >
+            <Box ml={'1px'} mt={'-5px'} display={'flex'}>
+              <USDTIcon size={'13px'} />
+            </Box>
+            <Box ml={'-4px'} mt={'5px'} display={'flex'}>
+              <USDTIcon size={'13px'} />
+            </Box>
+          </Box>
+        </Box>
+      </Box>
+      <Box
+        mt={'40px'}
+        fontSize={'20px'}
+        fontWeight={'700'}
+        textAlign={'left'}
+        color={'#ebecf2'}
+      >
+        Leverage assets you believe in
+      </Box>
+      <Box mt={'23px'} display={'flex'} gridGap={'24px'} flexWrap={'wrap'}>
+        <Box
+          flex={'1'}
+          sx={{ minWidth: { xs: '55%', sm: '35%', md: '20%' } }}
+          bgcolor={'#232734'}
+          p={'24px'}
+          borderRadius={'12px'}
+        >
+          <Box color={'#696c80'} fontSize={'14px'}>
+            Total Supply
+          </Box>
+          <Box fontSize={'24px'} color={'white'}>
+            $19.2M
+          </Box>
+        </Box>
+        <Box
+          flex={'1'}
+          sx={{ minWidth: { xs: '55%', sm: '35%', md: '20%' } }}
+          bgcolor={'#232734'}
+          p={'24px'}
+          borderRadius={'12px'}
+        >
+          <Box color={'#696c80'} fontSize={'14px'}>
+            Total Borrowed
+          </Box>
+          <Box fontSize={'24px'} color={'white'}>
+            $19.2M
+          </Box>
+        </Box>
+        <Box
+          flex={'1'}
+          sx={{ minWidth: { xs: '55%', sm: '35%', md: '20%' } }}
+          bgcolor={'#232734'}
+          p={'24px'}
+          borderRadius={'12px'}
+        >
+          <Box color={'#696c80'} fontSize={'14px'}>
+            Liquidity
+          </Box>
+          <Box fontSize={'24px'} color={'white'}>
+            $19.2M
+          </Box>
+        </Box>
+        <Box
+          flex={'1'}
+          sx={{ minWidth: { xs: '55%', sm: '35%', md: '20%' } }}
+          bgcolor={'#232734'}
+          p={'24px'}
+          borderRadius={'12px'}
+        >
+          <Box color={'#696c80'} fontSize={'14px'}>
+            Pull Utilization
+          </Box>
+          <Box fontSize={'24px'} color={'white'}>
+            $19.2M

```

Comment:
Why is this hardcoded?

Reply:
updated the value on top of zeplin mockup.